ext/afform/core/sql/upgrade_1001.sql

```diff
@@ -0,0 +1,46 @@
+-- +--------------------------------------------------------------------+
+-- | Copyright CiviCRM LLC. All rights reserved.                        |
+-- |                                                                    |
+-- | This work is published under the GNU AGPLv3 license with some      |
+-- | permitted exceptions and without any warranty. For full license    |
+-- | and copyright information, see https://civicrm.org/licensing       |
+-- +--------------------------------------------------------------------+
+--
+-- Generated from schema.tpl
+-- DO NOT EDIT.  Generated by CRM_Core_CodeGen
+--
+-- /*******************************************************
+-- *
+-- * Clean up the existing tables - this section generated from drop.tpl
+-- *
+-- *******************************************************/
+
+SET FOREIGN_KEY_CHECKS=0;
+
+DROP TABLE IF EXISTS `civicrm_afform_submission`;
+
+SET FOREIGN_KEY_CHECKS=1;
```

Comment:
I believe these lines are redundant with the conditional in `function upgrade_1001()` and as a general rule I think we should avoid these "loaded gun" self-destruct statements.

Reply:
@colemanw Yeah, that bit was just a blind restoration. But I think you're right. Updated.